### PR TITLE
release: version 2.12.0

### PR DESCRIPTION
**Pull request by nitayneeman**

### CODE REVIEW

**Review by sivanharel — APPROVED** [review]

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_507/

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
+# [2.12.0](https://github.com/wix/wix-ui-tpa/compare/v2.11.1...v2.12.0) (2020-12-02)
+
+
+### Bug Fixes
+
+* **Button:** Fix prefix and suffix alignments ([459580f](https://github.com/wix/wix-ui-tpa/commit/459580f2d9bb372a852dff2b1b89323ca929c1f2))
+* **TextField:** keep stylable state syntax ([20e9fb8](https://github.com/wix/wix-ui-tpa/commit/20e9fb8ef4a93bf6ae27f6c44de57e35c25a530c))
```
Comment (jonathanadler) [inline CHANGELOG.md:7]: This line can be removed actually.
It's a fix I made for the feature below 👇

Comment (jonathanadler) [reply CHANGELOG.md:7]: So it's not really a bug fix

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
+# [2.12.0](https://github.com/wix/wix-ui-tpa/compare/v2.11.1...v2.12.0) (2020-12-02)
+
+
+### Bug Fixes
+
+* **Button:** Fix prefix and suffix alignments ([459580f](https://github.com/wix/wix-ui-tpa/commit/459580f2d9bb372a852dff2b1b89323ca929c1f2))
+* **IconButton:** fix stylable warning ([492a955](https://github.com/wix/wix-ui-tpa/commit/492a955cb025eee3b455eb4b6da8c35b6e771af5))
```
Comment (jonathanadler) [inline CHANGELOG.md:7]: This can be removed again 😁  sorry...

**Review by jonathanadler — APPROVED** [review]

**Pull request merged**